.buildkite/dagster-buildkite/dagster_buildkite/steps/dagster.py

```diff
@@ -117,3 +118,25 @@ def build_graphql_python_client_backcompat_steps() -> List[CommandStep]:
         )
         .build()
     ]
+
+
+def skip_mysql_if_no_changes_to_dependencies(dependencies: List[str]):
+    if not is_feature_branch():
+        return None
+
+    for dependency in dependencies:
+        if python_packages.get(dependency) in python_packages.with_changes:
+            return None
+
+    return "Skip unless mysql schemas might have changed"
+
+
+def skip_graphql_if_no_changes_to_dependencies(dependencies: List[str]):
+    if not is_feature_branch():
+        return None
+
+    for dependency in dependencies:
+        if python_packages.get(dependency) in python_packages.with_changes:
+            return None
+
+    return "Skip unless GraphQL schemas might have changed"
```

Comment:
this should just be one `skip_if_no_changes_to_dependencies` function that includes the packages in the skip message. Having the target in the skip message is not really useful

Reply:
Yeah I have a simplifying refactor in mind for all of the skip logic coming up in a future PR.